<?xml version="1.0"?>
<launch>
  
  	<!-- node that updates calibration urdf with results -->
  	<node type="update_cob_calibration_urdf.py" pkg="cob_auto_calibrate" name="$(anon update_cob_calibration_urdf)" output="screen">
  		<param name="urdf_in" value="$(find cob_hardware_config)/$(env ROBOT)/calibration/calibration.urdf.xacro" />
  		<param name="urdf_out" value="$(find cob_hardware_config)/$(env ROBOT)/calibration/calibration.urdf.xacro_updated" />
        <param name="initial_system" value="$(find cob_auto_calibrate)/config/params_v0/system.yaml" />
        <param name="calib_system" value="/tmp/cal/result_step_3.yaml" />
        
        <!-- set to True for debug output -->
        <param name="debug" value="True" />
 	</node>
 
</launch>
